### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by VenkyOS**

### CODE REVIEW

**Review by mukul-seagate11 — APPROVED** [review]
looks good

**Review by shailesh-vaidya — APPROVED** [review]
Looks good.

**Review by pujamudaliar — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ You can report instances of abusive, harassing, or otherwise unacceptable behavi
 
 **Core Functionality:** For basic development and testing of CORTX, you only need the motr and S3 submodules.
 
-1. [CORTX Motr](https://github.com/Seagate/cortx-motr/blob/dev/doc/Quick-Start-Guide.rst)
+1. [CORTX Motr](https://github.com/Seagate/cortx-motr/blob/main/doc/Quick-Start-Guide.rst)
```
Comment (huanghua78) [inline CONTRIBUTING.md:30]: *[defect]*

Usually, in the URL link, we don't include 'branch' name. For this one, we need:
```
1. [CORTX Motr](/doc/Quick-Start-Guide.rst)
```

This rule should be used in all places in this doc. Please update all the links in the  this file and in "doc/architecture.md" file.

**Review by huanghua78 — CHANGES_REQUESTED** [review]
Changes needed.

File: CONTRIBUTING.md (hunk 1 above)
Comment (VenkyOS) [reply CONTRIBUTING.md:30]: I tried doing that across documents but the problem is they are breaking.

Comment (huanghua78) [reply CONTRIBUTING.md:30]: Sorry, this is a URL pointing to cortx-motr.
Yes, it should include a full link.

Comment (huanghua78) [reply CONTRIBUTING.md:30]: This looks fine.

**Review by huanghua78 — APPROVED** [review]

**Pull request merged**